docs/node.md

```diff
@@ -69,7 +69,7 @@ Kubernetes from the node.
 ## Node Management
 
 Unlike [Pods](pods.md) and [Services](services.md), a Node is not inherently
-created by Kubernetes: it is either created from cloud providers like GCE,
+created by Kubernetes: it is either created from cloud providers like Google Container Engine,
```

Comment:
No didnt mean to change it, thanks. Ive fixed it so it now says Google Compute Engine. Should be good to go now.